packages/lib-sourcify/src/lib/types.ts

```diff
@@ -36,29 +38,99 @@ export interface MetadataSources {
   };
 }
 
+export interface MetadataOutput {
+  abi: Abi;
+  devdoc: {
+    author?: string;
+    details?: string;
+    errors?: {
+      [index: string]: {
+        details?: string;
+      };
+    };
+    events?: {
+      [index: string]: {
+        details?: string;
+        params?: any;
+      };
+    };
+    kind: 'dev';
+    methods: {
+      [index: string]: {
+        details?: string;
+        params?: any;
+        returns?: any;
+      };
+    };
+    stateVariables?: any;
+    title?: string;
+    version?: number;
+  };
```

Comment:
Let's refactor this and `userdoc` out into exportable `Devdoc` and `Userdoc` types. I guess there are no NatSpec types being maintained, so we should maintain it ourselves and keep it reusable.